### PR TITLE
wg: add example to show current configuration of an interface

### PR DESCRIPTION
**Pull request by mfrw**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

PS: I am not a french speaker, i took help from Google Translate.

### CODE REVIEW

File: pages.fr/linux/wg.md [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
 
 `wg genkey`
 
-- Générer une clé publique à partir d'une clé privée :
+- Générer une clé publique à partir d'une clé privée:
 
 `wg pubkey < {{chemin/vers/clé_privée}} > {{chemin/vers/clé_publique}}`
 
-- Générer une clé publique et privée :
+- Générer une clé publique et privée:
```
Comment (navarroaxel) [inline pages.fr/linux/wg.md:18]: this in French, the space before the `:` is specific French grammar.

Comment (mfrw) [reply pages.fr/linux/wg.md:18]: Ohh sure :) my bad :(

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
How did you translate this? I'm going to @nicokosi @patricedenis just to make sure the translation is correct.

**Comment by mfrw** [comment]
> How did you translate this? I'm going to @nicokosi @patricedenis just to make sure the translation is correct.

I used google translate.

**Review by nicokosi — APPROVED** [review]
👌 for me. Thanks @mfrw! 👍

**Pull request merged**